ext/TOML/src/parser.jl

```diff
@@ -54,13 +55,7 @@ Base.position(p::Parser) = Int(position(p.input))+1
 Base.write(p::Parser, x) = write(p.charbuffer, x)
 Base.read(p::Parser) = p.currentchar = read(p.input, Char)
 
-NONE() = nothing
-NONE(::Type{T}) where {T} = nothing
-SOME(v::T) where {T} = v
-NONE(::Type{String}, p::Parser) = (take!(p.charbuffer); nothing)
-SOME(::Type{String}, p::Parser) = String(take!(p.charbuffer))
-isnull(x) = x === nothing
-get(x) = (@assert !isnull(x); x)
+isnull(x) = x === nothing  # works with older versions of Julia as isnothing() only introduced in 1.1
```

Comment:
We should change this to just use `x === nothing` inline since that currently works better with the compiler, but that can be done in another PR.